src/addons.js

```diff
@@ -77,6 +82,12 @@ async function updateAddon(settings, netlifyApiToken) {
     }),
   })
 
+  const data = await response.json()
+
+  if (response.status === 422) {
+    throw new Error(`Error ${JSON.stringify(data)}`)
+  }
+
   return response
```

Comment:
just realized that we're not passing back the json formatted response like get/create. We could potentially just return data instead of returning response. Looking at how we're doing for create/update in CLI side, returning `data` here doesn't look like causes issues but we may want to double check with that.

https://github.com/netlify/cli/blob/6a960dc1ecf0e47134d195c3b7b38f4b503c2802/src/commands/addons/create.js#L143
https://github.com/netlify/cli/blob/6a960dc1ecf0e47134d195c3b7b38f4b503c2802/src/commands/addons/config.js#L171